### PR TITLE
(SIMP-3062) Allow FakeCA to take command line host

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Can use a different keydist target by setting an environment variable
* Can set the host that you wish to target at the command line (togen
  will be ignored)
* Updated the README
* Ensure that chowns are only done if the running user is 'root'

SIMP-3062 #comment Mods to FakeCA to allow command line host entries

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.1.9
```
Comment (jeefberkey) [inline .ruby-version]: Why now?

Comment (kendall-moore) [inline .ruby-version]: Nothing inside of these 4 commits modifies Ruby code.  If a version bump is needed, this is the wrong place.

**Comment by jeefberkey** [comment]
Can I use this from other directories besides the FakeCA directory?

It'd be cool to just run `/var/simp/environments/simp/FakeCA/gencerts_nopass.sh auto client.simp.test` and that's it

File: .ruby-version (hunk 1 above)
Comment (op-ct) [inline .ruby-version]: @kendall-moore: this isn't a version bump―the [`.ruby-version`](https://rvm.io/workflow/projects#project-file-ruby-version) file is used by RVM (and rbenv) to switch to that version of ruby (this case, matching our targets' AIO).

It's probably a good idea to maintain the file.  But if we go this route, we should do it deliberately  it across all the repositories (and keep the corresponding `PUPPET_VERSION` matrix in the `.travis.yml` up-to-date).

**Comment by trevor-vaughan** [comment]
@jeefberkey Yeah, you can run this from anywhere on the system. The path is based on the path of the script itself and everything is relative to that.

**Comment by nick-markowski** [comment]
@trevor-vaughan I can't run this outside of the FakeCA directory

**Comment by trevor-vaughan** [comment]
Making this runnable from anywhere

File: .ruby-version (hunk 1 above)
Comment (jeefberkey) [inline .ruby-version]: Creating this file also codifies some undocumented developer knowledge required to develop on SIMP.

**Comment by trevor-vaughan** [comment]
Whoever reviews this, please make sure that the commit message is correctly set. The original message is not correct.

**Comment by jeefberkey** [comment]
@trevor-vaughan you can edit the name of a PR at any time to closer reflect the contained changes

**Pull request merged**